### PR TITLE
Design task/task 3 - First view

### PR DESCRIPTION
**Pull request by Conormc98**
Pull request for the first of the three views.

### CODE REVIEW

File: kjourney/src/app/app.component.html [hunk 1]
```diff
@@ -9,17 +9,18 @@
...
   </header>
 
   <a routerLink="home" routerLinkActive="active">Home</a> <!-- TODO: Replace with navbar -->
   <a routerLink="login">Login</a>
+  <a routerLink="band" routerLinkActive="active">Band</a>
 
   <router-outlet></router-outlet>
 
-  <footer>
+  <!--<footer>
```
Comment (ChrisThompsonK) [inline kjourney/src/app/app.component.html:21]: Should we remove this instead of just comment it out?

File: kjourney/src/app/app.component.html [hunk 2]
```diff
@@ -9,17 +9,18 @@
...
     <p>© Copyright 2019</p>
-  </footer>
+  </footer>-->
```
Comment (ChrisThompsonK) [inline kjourney/src/app/app.component.html:23]: ^^

File: kjourney/src/app/band/band.component.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+			</tr>
+        </td>
+        <td>
+			<tr class="myRow" id="pop-over" *ngFor="let role of roles">
+				<button class="button1">{{ role.name }}</button>
+			</tr>
+        </td>
+        <td>
+			<tr class="myRow" id="pop-over" *ngFor="let role of roles">
+				<button class="button1">{{ role.name }}</button>
```
Comment (ChrisThompsonK) [inline kjourney/src/app/band/band.component.html:30]: Do we need 2 of the same button?

File: kjourney/src/app/band/band.component.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    </td>
+    <td align="middle" style="width: 40%;">
+        <h1>Roles:</h1>
+        <div style="text-align: center; background-color: white; overflow: auto; border: 1px solid black; height: 300px; width: 100%; float: middle;">Roles <br>
+        <tr float="middle" class="myRow" id="pop-over" *ngFor="let role of roles">
+            <button class="button1">{{ role.name }}</button>
+        </tr>
+        </div>
+    </td>
+</table> -->
```
Comment (ChrisThompsonK) [inline kjourney/src/app/band/band.component.html:56]: Are we going to be using this? If not we should remove

File: kjourney/src/app/band/band.component.html (hunk 3 above)
Comment (Conormc98) [reply kjourney/src/app/band/band.component.html:30]: @ShaneKainos

File: kjourney/src/app/band/band.component.html (hunk 4 above)
Comment (Conormc98) [reply kjourney/src/app/band/band.component.html:56]: @ShaneKainos

**Review by ChrisThompsonK — CHANGES_REQUESTED** [review]
I've left some comments to address but otherwise this looks good to me

File: kjourney/src/app/app.module.ts [hunk 5]
```diff
@@ -3,24 +3,36 @@ import { NgModule } from '@angular/core';
...
+import { BandComponent } from './band/band.component';
 
 @NgModule({
   declarations: [
     AppComponent,
     MainComponent,
-    LoginComponent
+    LoginComponent,
+    ViewTabsComponent,
+    View1Component,
```
Comment (micalexis) [inline kjourney/src/app/app.module.ts:20]: what is going on here? what is the agreed view design you guys agreed on? remove all others!

File: kjourney/src/app/view1/view1.component.ts [hunk 6]
```diff
@@ -0,0 +1,260 @@
+import {Component, OnInit, ViewEncapsulation} from '@angular/core';
```
Comment (micalexis) [inline kjourney/src/app/view1/view1.component.ts:1]: this whole approach needs a rethink! but for now leave a comment in the code to look into this again!

File: kjourney/src/app/view2/view2.component.html [hunk 7]
```diff
@@ -0,0 +1 @@
+<p>view2 works!</p>
```
Comment (micalexis) [inline kjourney/src/app/view2/view2.component.html]: remove!

**Review by micalexis — CHANGES_REQUESTED** [review]
address comments!!!

File: kjourney/src/app/app.module.ts (hunk 5 above)
Comment (Conormc98) [reply kjourney/src/app/app.module.ts:20]: Removed all the other views and the tabs, leaving only the first view which will be rethought and reimplemented early next sprint

**Pull request closed without merging**